content/concepts/introduction-to-webpack.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Introduction
+---
+
+*Webpack* is a module system for modern Javascript applications.
+
+In [modular programming](https://en.wikipedia.org/wiki/Modular_programming), developers break programs up into discrete chunks of functionality.
+This has a number of benefits.
+Each module has a smaller surface area for testing than a full program, making verification and bug-hunting easier.
+Well-written modules provide solid abstractions and encapsulation boundaries, so that each module has a coherent design and a clear purpose within the overall application.
+
+Node has supported modular programming almost since its inception.
+On the web, however, support for modules has been slow to arrive.
+Multiple tools exist that support modular Javascript on the web, with a variety of benefits and limitations.
+Webpack builds on lessons learned from these systems.
+
+In a Webpack application, each _module_ contains all of the code necessary to execute a single aspect of the application.
+Modules do not need to be self-contained.
+Code provided by one module can be reused in a second module: the first module _depends on_ the second module.
+
+In Webpack, dependencies can be expressed in many ways. A few examples are:
+
+<!-- * A module written in Javascript can depend on another module using the CommonJS `var module = require(modulePath)` function, or using AMD's `define([modulePath], function(module) {})` function.
+* A module written in ES6 can depend on another module using ES6's `import module from 'modulePath'` syntax.
+* A module written in CSS can depend on another module using CSS `@import` rules.
+ -->
+
+* A Javascript `require()` statement
+* An ECMAScript2015 `import` statement
```

Comment:
Maybe it's good to mention webpack 1 requires a specific loader for this, though that can be skipped as well.